dateparser_cli/fasttext_manager.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+import urllib.request
+import os
+import logging
+
+from .utils import date_parser_model_home, check_data_model_home_existance
+from .exceptions import FastTextModelNotFoundException
+
+
+def fasttext_downloader(model):
+    check_data_model_home_existance()
+
+    model_url = {
+        "small": "https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.ftz",
+        "large": "https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.bin"
+    }
+
+    if model in model_url:
+        model_name = model
+    else:
+        message = "Couldn't find a model called \"{}\". Supported models are: {}".format(
+            model, "in ".join(model, model_url.keys())
+        )
+        raise FastTextModelNotFoundException(message)
+
+    models_directory_path = os.path.join(date_parser_model_home, (model_name + ".bin"))
+
+    if not Path(models_directory_path).is_file():
+        model_url = model_url[model_name]
+        logging.info("Downloading model {}...".format(model_name))
```

Comment:
Not sure, but what do you think about also adding a message when the model was downloaded? It might help with debugging if this is the last message in the log.

Reply:
Like a print message about the download? or something else I could not understand as the download information is already logged.

Reply:
@lopuhin, I think what you mention is some lines below: 

```
logging.info("dateparser-download: The model is already downloaded")
```